core/src/main/web/app/mainapp/components/notebook/outputdisplay/outputdisplayfactory-service.js

```diff
@@ -82,12 +82,27 @@
           });
         },
         link: function(scope, element, attrs) {
+          var tagstofilter = ['applet', 'base', 'basefont', 'body', 'frame', 'frameset', 'head', 'html',
```

Comment:
well, this just seems so simple to bypass that I am not even sure why we want it.